config/themes.json

```diff
@@ -26,6 +26,7 @@
         "CloseIconFontSize":      "18",
 
         "MicroWinLogoSize": "10",
+        "MicrowinCheckBoxMargin": "-10,5,0,0",
```

Comment:
This's a temporary fix when I made it, as it "resets" the left padding to the left by doing some negative margins. Needs a better solution for sure 👍

Reply:
yeah u are right, I really don't have time rn but ill look into it some time, thanks for informing me :)

Reply:
Anyways, a UI Refractor is also needed for Microwin. Especially the fact that there are custom margins/paddings almost everywhere with low standardisation. I'll save this for another time/anotheone to fix